pkgs/applications/science/biology/migrate/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, fetchurl, zlib, gcc, glibc, openmpi }:
+
+stdenv.mkDerivation rec {
+  version = "3.7.2";
+  pname = "migrate";
+
+  src = fetchurl {
+    url = "https://peterbeerli.com/migrate-html5/download_version3/${pname}-${version}.src.tar.gz";
+    sha256 = "0mb8b0jrjb83bp2f6if2gdwz5f0nbysc1vchn5bfa4jhxfgscr53";
```

Comment:
When building, I got:
```
hash mismatch in fixed-output derivation '/nix/store/9zh4gafq7wwswpqi7lkm65wsbzjh5fqz-migrate-3.7.2.src.tar.gz':
  wanted: sha256:0mb8b0jrjb83bp2f6if2gdwz5f0nbysc1vchn5bfa4jhxfgscr53
  got:    sha256:1p2364ffjc56i82snzvjpy6pkf6wvqwvlvlqxliscx2c303fxs8v
```

Reply:
Weird...
https://peterbeerli.com/migrate-html5/download_version3/
```
 migrate-3.7.2.src.tar.gz        01-Apr-2019 07:07   25M
```
It is actually 1p2364ffjc56i82snzvjpy6pkf6wvqwvlvlqxliscx2c303fxs8v